lisa/tools/wget.py

```diff
@@ -44,6 +44,11 @@ def get(
         matched_result = self.__pattern_path.match(command_result.stdout)
         if matched_result:
             download_file_path = matched_result.group("path")
+        else:
+            raise LisaException(
```

Comment:
It still can be improved, since it doesn't give the reason why download file path doesn't exist. What I guess is,

f"cannot find file path in output of '{run_command}', it may cause by download failed."